From efc9e80f99b09defb7caa2fd17802ea2dcd7b875 Mon Sep 17 00:00:00 2001
From: Dante Russo <drusso@codeaurora.org>
Date: Fri, 22 Apr 2016 14:19:33 -0700
Subject: [PATCH 04/11] msm8974: Default apn ip type to ipv4
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

If a apn ip type is not found for a agps
data connection, default to using ipv4
instead of invalid.

CRs-fixed: 1007417

Bug: 28986221

Change-Id: Icc71de1fab3fd5967901c39a5c11eaea17b7766c
Signed-off-by: D. Andrei Măceș <Andrei.Maces@alumni.nd.edu>
---
 msm8974/loc_api/libloc_api_50001/loc.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/msm8974/loc_api/libloc_api_50001/loc.cpp b/msm8974/loc_api/libloc_api_50001/loc.cpp
index f6630d5..2f2cdca 100644
--- a/msm8974/loc_api/libloc_api_50001/loc.cpp
+++ b/msm8974/loc_api/libloc_api_50001/loc.cpp
@@ -832,7 +832,7 @@ static int  loc_agps_open_with_apniptype(const char* apn, ApnIpType apnIpType)
             bearerType = AGPS_APN_BEARER_IPV4V6;
             break;
         default:
-            bearerType = AGPS_APN_BEARER_INVALID;
+            bearerType = AGPS_APN_BEARER_IPV4;
             break;
     }
 
-- 
2.17.1

